Dockerfile

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) Meta, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+FROM nvidia/cuda:11.1.1-cudnn8-devel-ubuntu18.04
+
+# Installing the required packages
+RUN apt update -y
+RUN apt install -y git curl
+
+# Installing Anaconda
+WORKDIR /root
+RUN curl https://repo.anaconda.com/archive/Anaconda3-2021.05-Linux-x86_64.sh -o anaconda_installer.sh
+RUN bash anaconda_installer.sh -b -p
+ENV PATH="/root/anaconda3/bin:$PATH"
+
+# Installing recommmended pre-requirements
+RUN conda install pytorch torchvision torchaudio -c pytorch-lts -c nvidia
```

Comment:
should we specify torch version? or just assume we work with highest torch number?

Reply:
Yeah, good point. Should we set it to the same version as the install requirements?